src/js/stores/AppsStore.js

```diff
@@ -10,6 +11,24 @@ function removeApp(apps, appId) {
   }).value();
 }
 
+function removeTask(tasks, relatedAppId, taskId) {
+  return lazy(tasks).reject(function (task) {
+    return task.id === taskId && task.appId === relatedAppId;
+  }).value();
+}
+
+function processCurrentApp(currentApp) {
```

Comment:
As this is, essentially, a "constructor" method, I wonder if it'd be a nice idea to do something more ES6 OOP, along the lines of:

```
class CurrentApp { 
  constructor(app) {
    this.tasks = app.tasks ? app.tasks : [];
  }
};
```

and then later ( line 49 )

```
AppsStore.currentApp = new CurrentApp(action.data);
```

That's just an idea. In general I'd be fine with just renaming the function to `initializeCurrentApp` as `process` has a different meaning... 

But feel free to ignore this comment.